presto-native-execution/presto_cpp/presto_protocol/README.md

```diff
@@ -24,6 +24,10 @@ Dependencies to Install
    You will need to note the path to the installed executable and add this to your PATH.
    Something like `/Users/adutta/Library/Python/3.8/bin`.
 
+Protocol Code Generation:
+
+   Run in terminal `make presto_protocol`
```

Comment:
I think you need to specify PRESTO_HOME. Otherwise it will default to ~/presto
PRESTO_HOME=/Users/jtan6/git-repos/presto make presto_protocol

Reply:
I added this requirement to README